.changeset/config.json

```diff
@@ -1,21 +1,23 @@
 {
-  "$schema": "https://unpkg.com/@changesets/config@3.0.0/schema.json",
-  "changelog": [
-    "@changesets/changelog-github",
-    {
-      "repo": "adobe/spectrum-css"
-    }
-  ],
-  "commit": false,
-  "fixed": [],
-  "linked": [],
-  "access": "public",
-  "baseBranch": "main",
-  "updateInternalDependencies": "minor",
-  "useCalculatedVersion": true,
```

Comment:
Sorry for the prettier changes here! The update I made here is removing this `useCalculatedVersion` setting which is only valid in the snapshot section. I then added the snapshot section so we can activate that option when we're rolling snapshot releases.